src/js/tech/tech.js

```diff
@@ -234,10 +232,10 @@ class Tech extends Component {
       }
     };
 
-    tracks.addEventListener('change', textTracksChanges);
+    tracks.addEventListener('change', Fn.bind(this, textTracksChanges));
 
     this.on('dispose', Fn.bind(this, function() {
-      tracks.removeEventListener('change', textTracksChanges);
+      tracks.removeEventListener('change', Fn.bind(this, textTracksChanges));
```

Comment:
You could bind this when textTracksChanges is created so you don't have to do it when both adding and removing it.